src/hrtf/easy.c

```diff
@@ -15,7 +16,7 @@
  *
  */
 
-MYSOFA_EXPORT struct MYSOFA_EASY* mysofa_open(const char *filename, float samplerate, int *filterlength, int *err)
+MYSOFA_EXPORT struct MYSOFA_EASY* mysofa_open_default(const char *filename, float samplerate, int *filterlength, int *err, bool applyNorm)
```

Comment:
Please define struct MYSOFA_EASY* mysofa_open_default as a static function, not as exported.